pcsx2/ImGui/ImGuiOverlays.cpp

```diff
@@ -152,6 +154,16 @@ __ri void ImGuiManager::DrawPerformanceOverlay(float& position_y, float scale, f
 			else
 				text.append_format(" ({:.0f}%)", target_speed * 100.0f);
 		}
+
+		if (GSConfig.OsdShowVersion)
+			{
```

Comment:
The body of the if statement has an extra tab.

Reply:
Fixed